RadioTransmission_ST/RadioTransmission_ST.py

```diff
@@ -23,16 +23,18 @@ def __modulateSignal():
     modulator = Modulator(__CARRIER_FREQ, __SYMBOL_LENGTH_IN_BITS,  __SAMPLING_RATE)
     return modulator.modulate(__BITS)
 
+
 def __demodulateSignal(signal):
     demodulator = Demodulator(__CARRIER_FREQ, __SYMBOL_LENGTH_IN_BITS, __SAMPLING_RATE)
     return demodulator.demodulate(signal)
 
-def __modulateAndDemodulate(snr=None, attenuation=1):
+def __modulateAndDemodulate(snr=None, attenuation=1, carrFreq=None):
```

Comment:
co to i po co to? nie ma takiego argumentu

Reply:
takie gówno wcommitowałem? ⭕️

Reply:
:O

Reply:
nooooooo